versions.json

```diff
@@ -278,9 +260,10 @@
         "rpm_dependencies": ["xz-libs >= 5.2.2"]
     },
     "libs3": {
+        "git_repository": "https://github.com/alanking/libs3",
         "enable_sha": true,
-        "commitish": "c0e278d280cde08fb588d82dcb125198852f2bc9",
-        "version_string": "c0e278d2",
+        "commitish": "openssl3",
+        "version_string": "openssl3",
```

Comment:
Is this what we want for the commitish and version string?

I'm guessing this is a placeholder for the real thing because it points to your fork of libs3?

Reply:
Yep, once we merge https://github.com/irods/libs3/pull/22, we can update the URL and the sha.

Reply:
Done